.github/workflows/publish.yml

```diff
@@ -274,13 +208,86 @@ jobs:
           make build-ui
       - name: Build
         run: |
-          python3 setup.py sdist bdist_wheel
-
+          python3 setup.py sdist
```

Comment:
why are we removing bdist_wheel here?

Reply:
we are building it in the py310 build. this is just for the source distribution